### DISCUSSION TITLE
About the website of this course

### DISCUSSION THREAD

**Discussion by haoran-ni** [mrdbourke/pytorch-deep-learning#373] [General]
Hi! This question may seem irrelevant to the content of this course, but I am curious to know how is the website https://www.learnpytorch.io/ constructed from this repo? I know that github can automatically complie the README.md file into the website, but the course website looks different from the default theme. I really like the theme of this website, especially that sidebar. Is it possible that anyone could tell me how to make such a website from a github repo?

**Comment by humanfriend22** [comment]
If you scroll _all_ the way down on the homepage, you can see that the theme is [Material For MkDocs](https://squidfunk.github.io/mkdocs-material/).

**Reply by haoran-ni** [reply]
Got it! Thank you!
